test/cypress/support/commands.ts

```diff
@@ -1,5 +1,17 @@
 import '@testing-library/cypress/add-commands'
 
+declare global {
```

Comment:
Why is this declaration like so? Is it because we need to reuse the interface across different files without importing it? In that case, it might be worth using a `d.ts`

Reply:
>Is it because we need to reuse the interface across different files without importing it?

Yes.

>In that case, it might be worth using a d.ts

Because it's a very small file with only a few definitions, I think it's more convenient to put it directly it in the definition file.